plugins/ecs-bridge/engine/engine.go

```diff
@@ -0,0 +1,234 @@
+// Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
How about `BridgePlugin` or even `Plugin`? The actions taken are of the plugin itself so it makes some sense and is slightly more contextual than Engine (I'm not the greatest at coming up with names).